docs/platforms/Swift-Package-Manager.md

```diff
@@ -0,0 +1,645 @@
+
+14. Update your `plugin_name_ios.podspec` to point to new paths.
+```diff
+- s.source_files = 'Classes/**/*.{h,m}'
++ s.source_files = 'plugin_name_ios/Sources/plugin_name_ios/**/*.{h,m}'
+
+- s.public_header_files = 'Classes/**/*.h'
++ s.public_header_files = 'plugin_name_ios/Sources/plugin_name_ios/include/**/*.h'
+
+- s.module_map = 'Classes/cocoapods_plugin_name_ios.modulemap'
++ s.module_map = 'plugin_name_ios/Sources/plugin_name_ios/include/cocoapods_plugin_name_ios.modulemap'
+
+- s.resource_bundles = {'plugin_name_ios_privacy' => ['Resources/PrivacyInfo.xcprivacy']}
++ s.resource_bundles = {'plugin_name_ios_privacy' => ['plugin_name_ios/Sources/plugin_name_ios/PrivacyInfo.xcprivacy']}
+```
+
+15. Update getting of resources from bundle to use `SWIFTPM_MODULE_BUNDLE`
+```objc
+#if SWIFT_PACKAGE
+   NSBundle *bundle = SWIFTPM_MODULE_BUNDLE;
+ #else
+   NSBundle *bundle = [NSBundle bundleForClass:[self class]];
+ #endif
+ NSURL *imageURL = [bundle URLForResource:@"image" withExtension:@"jpg"];
+```
+  * Note: `SWIFTPM_MODULE_BUNDLE` will only work if there are actual resources (either [defined in the Package.swift](https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package#Explicitly-declare-or-exclude-resources) or [automatically included by Xcode](https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package#:~:text=Xcode%20detects%20common%20resource%20types%20for%20Apple%20platforms%20and%20treats%20them%20as%20a%20resource%20automatically)). Otherwise, it will fail.
+
+16. If your `plugin_name_ios/Sources/plugin_name_ios/include` directory only contains a `.gitkeep`, you'll want update your `.gitignore` to include the following:
+```gitignore
+!.gitkeep
+```
+
+Then run `flutter pub publish --dry-run` to ensure the `include` directory will be published.
+
+17. Verify plugin still works with CocoaPods
+    1. Disable Swift Package Manager
+      ```
+      flutter config --no-enable-swift-package-manager
+      ```
+    2. Run `flutter run` with the example app and ensure it builds and runs
+18. Verify plugin works with Swift Package Manager
+    1. Enable Swift Package Manager
+      ```
+      flutter config --enable-swift-package-manager
+      ```
+    2. Run `flutter run` with the example app and ensure it builds and runs
+    3. Open the example app in Xcode and ensure Package Dependencies show in the left Project Navigator
+
+19. Verify tests pass
+  * **If your plugin has Native unit tests (XCTest), make sure you also complete "Updating unit tests in plugin example app" below.**
+  * [Follow instructions for testing plugins](https://docs.flutter.dev/testing/testing-plugins)
+</details>
+
+<details>
+  <summary>Converting an existing Swift Flutter plugin to a Swift package</summary>
+
+### Converting an existing Swift Flutter Plugin to a Swift Package
+
+Replace `plugin_name` throughout this guide with the name of your plugin.
```

Comment:
```suggestion
Replace `plugin_name` throughout this guide with the name of your plugin. The below example uses `ios`, replace `ios` with `macos`/`darwin` as applicable.
```

Reply:
Updated (and removed this content from the second step)